### DISCUSSION TITLE
"Wobble" mod from McOsu

### DISCUSSION THREAD

**Discussion by anexistantperson** [ppy/osu#13123] [Ideas]
**Describe the new feature:** 
Something similar to the wobble mod from McOsu!
Its easy to misremember as being a clone of the transform mod, because of this it should probably stack on top of transform.

**Proposal designs of the feature:** 
Here's a video of someone playing with the mod: https://youtu.be/gezITUO_DKg

**Comment by GSculerlor** [comment]
Is it the same effect as Wiggle Mod?

**Comment by anexistantperson** [comment]
No. Wiggle makes everything vibrate. I don't know how to explain wobble just watch the video.

**Comment by Daisyuki** [comment]
> Is it the same effect as Wiggle Mod?

No this 1 u apply alot of s_shake and rotate the camera

**Comment by iSimpFor** [comment]
also have a variation of the mod without the wobble

rotate direction should also be random
